sources/packages/web/src/services/http/dto/Assessment.dto.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  AssessmentTriggerType,
+  ScholasticStandingStatus,
+  StudentAppealStatus,
+  StudentAssessmentStatus,
+} from "@/types";
+
+export const ASSESSMENT_ALREADY_IN_PROGRESS = "ASSESSMENT_ALREADY_IN_PROGRESS";
+
+export interface RequestAssessmentSummaryAPIOutDTO {
+  id: number;
+  submittedDate: Date;
+  status: StudentAppealStatus | ScholasticStandingStatus;
+  triggerType: AssessmentTriggerType;
+}
+
+export interface AssessmentHistorySummaryAPIOutDTO {
+  assessmentId: number;
+  submittedDate: Date;
+  triggerType: AssessmentTriggerType;
+  assessmentDate: Date;
+  status: StudentAssessmentStatus;
+  studentAppealId?: number;
```

Comment:
Why is Out DTO using the optional field marker? Am I missing something?

Reply:
_studentAppealId_ and _studentScholasticStandingId_ are optional. Depending on how the assessment is generated they can be present or not.
That said, I do not see why an OutDTO would not possibly have an optional parameter. Am I missing something?

Reply:
You are right. OUT dtos can absolutely have optional parameter. 👍